static/themes/default/css/plinth.css

```diff
@@ -113,4 +113,23 @@ footer {
 
```

Comment:
In the Mobile Clients section, give some space between the store images and the text.